utils/rollout.js

```diff
@@ -0,0 +1,111 @@
+const crypto = require('crypto')
+
+const generateUUID = () => {
+  return crypto.randomBytes(8).toString('hex')
```

Comment:
Does this guarantee time-based uniqueness?

Reply:
Or what about uniqueness in a distributed environment?

Reply:
Neither of those is a requirement for this usecase. We don't need rigorous randomness.

That said, `randomBytes` is described as `cryptographically strong` so I believe it must be safe in those scenarios as well, though I did not find a way to confirm it. 

Refer: https://nodejs.org/api/crypto.html#crypto_crypto_randombytes_size_callback